src/components/Containers/Header/Header.module.scss

```diff
@@ -55,10 +55,16 @@ $chain-color: $--color-white-1;
   .right {
     margin-left: 20px;
 
-    button {
-      &:not(:last-child) {
+    & > button {
```

Comment:
This will get mess up if we'll have more buttons in the header.
create div with `.tabsContainer` class and set this as the selector.

Reply:
done